### ISSUE TITLE
Cannot enable Keybase in Finder

### ISSUE THREAD

**Issue by Mezzle** [keybase/client#24725]
I've had this issue on 2 macs now, so I think it's likely something I'm doing wrong.

I'm installing via homebrew, and am getting an odd error trying to install (`kOSKextReturnInternalError`)

my log id: db1274bdcc73a19158cfd71c

I'm getting this error, but clicking "Open Anyway" does nothing

![CleanShot 2021-12-08 at 15 55 47](https://user-images.githubusercontent.com/570639/145240112-1fab1927-6df5-4225-a6e0-67b33c98506f.png)

**Comment by MarkWieczorek** [comment]
I have just updated to Monterey and finder integration no longer works for me either. In the past, I have solved this problem by manually installing the macFuse using the dmg from the web site. This time, however, this solution did not work.

In the keybase client app, I have clicked on the "enable keybase in finder" button, however, after clicking "yes, enable", the icon turns for 10 seconds, and then goes back to the same screen which asks me to "yes, enable". Restarts, logging out, etc, have not helped. 

This is the last straw for me. I am sorry to say goodbye to keybase as it used to be great software.
